shiny-ci-cd.Rmd

```diff
@@ -34,9 +34,72 @@ Generally speaking, a CI/CD pipeline related to an R package is comprised of the
 
 Most of these steps are implemented by default in Travis CI for an R package. In GitHub Actions, on the other hand, it is currently necessary to manually specify each of them.
 
+## GitHub Actions
+
+[GitHub Actions](https://docs.github.com/en/free-pro-team@latest/actions) is a service for running highly-customizable and flexible automated workflows, fully integrated with GitHub and very suitable to CI/CD pipelines.
+[Workflows](https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-syntax-for-github-actions) use `YAML` syntax and should be stored in the `.github/workflows` directory in the root of the repository.
+Workflows are constituted of jobs and each job is a set of steps to perform individual tasks, e.g. commands or actions.
+
+### Workflow steps
+
+A workflow should have an identifying `name` and an `on` section that indicates upon which events the workflow should be triggered. It should include at least one job and each job will have a set of steps fully specifying what to execute. Such steps can be a (predefined) action or a script, however, for the time being, GitHub Actions does not support a hierarchical, aggregated structure of actions. Being fully customizable, it is necessary to fully specify each step in the CI/CD pipeline explicitly.
+
+#### Setup
+
+- Checkout the source package from the repo, using `actions/checkout` provided by GitHub.
```

Comment:
```suggestion
- Checkout the source package from the current repo, using the `actions/checkout` provided by GitHub.
```

Reply:
If this step also includes ubuntu installation, we should mention it explicitly.

Reply:
It does not. I don't like the added article, will remove it on a later commit.